README.md

```diff
@@ -7,7 +7,7 @@ Tested with Terraform version : 0.9.5
 # Installation
 
 1. Download the cloud.ca Terraform provider binary for your OS from the [releases page](https://github.com/cloud-ca/terraform-provider-cloudca/releases).
-2. Copy the provider to the directory where Terraform is located.
+2. Copy the provider to the plugin directory `~/.terraform.d/plugins`.
```

Comment:
Which version of terraform are you using?

Reply:
`Terraform v0.11.1`

Reply:
If all is working as expected, we could also bump the Tested with v0.9.5 version on the README